code/src/java/translation/util/Tips.java

```diff
@@ -70,37 +75,49 @@ public static void generatePOT(File rootDirectory, String potFilename)
 	 * @param rootDirectory root of the directories to parse
 	 * @param filename the name of the filename to parse
 	 */
-	public static void generatePOT(File rootDirectory, String potFilename, String filename)
+	private static void generatePOT(File rootDirectory,
+									String potFilename,
+									String filename)
 	{
 		Set<String> tips = new HashSet<>();
 		// search for each filename in the sub directory of rootDirectory
 		if (rootDirectory.isDirectory())
 		{
 			FilenameFilter filter = new SpecificFilenameFilter(filename);
 			File[] subfiles = rootDirectory.listFiles();
-			for (int i = 0; i < subfiles.length; i++)
+			for (File subfile : subfiles)
 			{
-				if (subfiles[i].isDirectory())
+				if (subfile.isDirectory())
 				{
-					File[] tipsFiles = subfiles[i].listFiles(filter);
-					for (int j = 0; j < tipsFiles.length; j++)
+					File[] tipsFiles = subfile.listFiles(filter);
+					for (File tipsFile : tipsFiles)
 					{
-						log("Found {0}", tipsFiles[j]);
-						// for each non comment line of the file, put its content in a Set<String>
+						log("Found {0}", tipsFile);
+						// for each non comment line of the file, put its content in a
+						// Set<String>
```

Comment:
If you are going to wrap single line comments in your formatter, then perhaps they should be  converged to multi-line comments if they are that long